### ISSUE TITLE
BUG in Preview Zoom for image

### ISSUE THREAD

**Issue by edwinhuish** [kartik-v/bootstrap-fileinput#696]
Image out of proportion in preview zoom.

I have read the code, change as blew, and it can resolve the BUG.

```
defaultPreviewZoomSettings = {

        image: {width: "auto", height: "auto", 'max-width': "100%",'max-height': "100%"},
        html: {width: "100%", height: "100%", 'min-height': "480px"},
        text: {width: "100%", height: "100%", 'min-height': "480px"},
        video: {width: "auto", height: "100%", 'max-width': "100%"},
        audio: {width: "100%", height: "30px"},
        flash: {width: "auto", height: "480px"},
        object: {width: "auto", height: "100%", 'min-height': "480px"},
        pdf: {width: "100%", height: "100%", 'min-height': "480px"},
        other: {width: "auto", height: "100%", 'min-height': "480px"}
    };
```

**Comment by kartik-v** [comment]
These are defaults. You can set your own image preview zoom settings via the property [previewZoomSettings](http://plugins.krajee.com/file-input#option-previewzoomsettings)

**Comment by carter-thaxton** [comment]
Within 5 minutes of trying out bootstrap-fileinput, I ran into this image preview distortion issue.
It's frustrating to have to find this issue and override the defaults in order to customize the images to appear correctly.

I'm not sure of any application where the fixed defaults provided by @edwinhuish wouldn't be more appropriate.  Are you worried about breaking compatibility?  He went out of his way to improve this plugin in a very reasonable way.  My vote is to accept this patch.

**Comment by kartik-v** [comment]
Can you please share a JS Fiddle to test that this works for most scenarios? Default Settings should be set to one which works for most scenarios.

Settings can be individually configured as mentioned - as each user will anyway have his/her dimensions of images to upload and it will need to be styled as per their UX design and containers.

I have reopened this issue - so that this can be tested and improved if needed.

**Comment by kartik-v** [comment]
Resolved via 5edbd4d and [documented in the change log](https://github.com/kartik-v/bootstrap-fileinput/blob/master/CHANGE.md).
